Sources/TensorFlow/Loss.swift

```diff
@@ -86,6 +86,26 @@ public func categoricalHingeLoss<Scalar: TensorFlowFloatingPoint>(
     return max(Tensor(0), negative - positive + Tensor(1))
 }
 
+// Helper function for `losCoshLoss(predicted:expected:)`.
+@differentiable(wrt: x)
```

Comment:
FYI: You don't need to specify `wrt` all parameters are bring differentiated.

```suggestion
@differentiable
```